src/test/java/io/airlift/airline/ArgConv.java

```diff
@@ -0,0 +1,115 @@
+package io.airlift.airline;
+
+public class ArgConv {
```

Comment:
We do not use abbreviations (except for certain well known).

Maybe `ArgumentConversion`?

Also, I'd put it as an inner class of the test class, but first we should clean the existing tests. So keep it separate for now.